package.json

```diff
@@ -113,6 +113,7 @@
     "load-grunt-tasks": "3.5.2",
     "mocha": "3.2.0",
     "mocha-multi": "0.10.0",
+    "mock-stdin": "0.3.1",
```

Comment:
We can probably reuse the mock that we created for #746 and do not include this as a devDependencies.